include/core/CameraMatrix.hpp

```diff
@@ -0,0 +1,103 @@
+#ifndef CAMERA_MATRIX_H
```

Comment:
@m4gr3d we need to check if this is indeed part of the ARCore work, I don't think it is, I think this was moved or renamed at some point so we may be reintroducing something here we shouldn't

Reply:
Never mind, looking at the wrong project :)